server/src/test/java/org/elasticsearch/index/snapshots/blobstore/FileInfoTests.java

```diff
@@ -31,8 +32,7 @@
 import static org.hamcrest.Matchers.is;
 
 public class FileInfoTests extends ESTestCase {
-    private static final Version MIN_SUPPORTED_LUCENE_VERSION = org.elasticsearch.Version.CURRENT.minimumIndexCompatibilityVersion()
-        .luceneVersion();
+    private static final org.apache.lucene.util.Version MIN_SUPPORTED_LUCENE_VERSION = IndexVersion.MINIMUM_COMPATIBLE.luceneVersion();
```

Comment:
tiny nit: I think we can use `Version` instead of `org.apache.lucene.util.Version`, since we're not importing `org.elasticsearch.Version`